packages/next/package.json

```diff
@@ -108,6 +108,7 @@
     "http-status": "2.1.0",
     "path-to-regexp": "6.3.0",
     "qs-esm": "7.0.2",
+    "react-grid-layout": "1.5.2",
```

Comment:
should we copy or install the library? CC @JarrodMFlesch @jacobsfletch @DanRibbens